cli/tools/fmt.rs

```diff
@@ -687,6 +704,35 @@ fn get_resolved_json_config(
   builder.build()
 }
 
+fn get_resolved_yaml_config(
+  options: &FmtOptionsConfig,
+) -> pretty_yaml::config::FormatOptions {
+  use pretty_yaml::config::*;
+
+  let mut layout_options = LayoutOptions::default();
+  if let Some(use_tabs) = options.use_tabs {
+    layout_options.use_tabs = use_tabs;
+  }
+  if let Some(line_width) = options.line_width {
+    layout_options.print_width = line_width as usize;
+  }
+  if let Some(indent_width) = options.indent_width {
+    layout_options.indent_width = indent_width as usize;
+  }
+
+  let mut language_options = LanguageOptions::default();
```

Comment:
Can you initialize this like:

```rs
let language_options = LanguageOptions {
  // ...
};
````

...and list every property along with the defaults? (same with LayoutOptions) That way the defaults here never change here.